src/main/java/io/jenkins/plugins/artifact_manager_jclouds/JCloudsArtifactManager.java

```diff
@@ -39,6 +39,7 @@
 import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.Functions;
 import io.jenkins.plugins.artifact_manager_jclouds.BlobStoreProvider.HttpMethod;
+import io.jenkins.plugins.artifact_manager_jclouds.s3.S3BlobStore;
```

Comment:
We may not import anything from the `s3` package in this package unless and until we get rid of the jclouds dep.

Reply:
fixed